portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title> Gallery of My Favorite Photos</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <div class="gallery-images">
+            <img src="gallery/gallery-1.jpg"/>
```

Comment:
It's great practice to add an alt attribute to all img tags, since it provides information if the img can't load or the user is utilizing a screen reader.

https://www.w3schools.com/tags/att_img_alt.asp